### PR TITLE
Merge latest changes from duplicate repos

### PR DESCRIPTION
**Pull request by hallihan**
azure-databricks-streaming-analytics and reference-architectures/data/streaming_azuredatabricks each had changes

Not including CRS changes, they don't seem to work with the Census data.

Testing done: full end-to-end confirmed data flowing to CosmosDB and Log Analytics.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,14 +16,19 @@ A deployment for this reference architecture is available on [GitHub](https://gi
 
 2. Install [Docker](https://www.docker.com/) to run the data generator.
 
-3. Install [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest).
+3. Install [Azure CLI 2.0](/cli/azure/install-azure-cli?view=azure-cli-latest).
```
Comment (ckittel) [inline README.md:19]: how does this URL work on github readmes?  Or does this get pulled into docs somewhere?

File: README.md [hunk 2]
```diff
@@ -233,26 +247,50 @@ For this section, you require the Log Analytics workspace ID and primary key. Th
 
 2. Select a **standard** cluster mode.
 
-3. Set **Databricks runtime version** to **6.4 (includes Apache Spark 2.4.5 Scala 2.11)**
+3. Set **Databricks runtime version** to **5.5 LTS (includes Apache Spark 2.4.3, Scala 2.11)**
 
-4. Set **Python version** to **3**.
+4. Set **Python version** to **2**.
```
Comment (ckittel) [inline README.md:252]: I don't have enough context here, so sorry for the ignorance, but just want to be sure we're going in the "right direction" with this... that somehow we aren't overwriting new with old.  Looks like we're rolling back to an older version 5.5 vs 6.4 (which might be just fine! -- just double checking)

File: azure/AzureDataBricksJob/src/main/java/com/microsoft/pnp/GeoFinder.java [hunk 3]
```diff
@@ -71,26 +68,18 @@ public static GeoFinder createGeoFinder(URL shapeFileUrl) throws IOException {
...
                     .getName());
             return new GeoFinder(featureSource, filterFactory, propertyName);
-        } catch (IOException ioe) {
-            logger.error(String.format("Error loading Geospatial data from %s", shapeFileUrl), ioe);
-            throw ioe;
-        } catch (FactoryException fe) {
-            logger.error(String.format("Error loading Geographic Coordinate System"), fe);
-            throw new IOException(fe);
+        } catch (IOException ex) {
+            logger.error(String.format("Error loading Geospatial data from %s", shapeFileUrl));
```
Comment (ckittel) [inline azure/AzureDataBricksJob/src/main/java/com/microsoft/pnp/GeoFinder.java:81]: Nit: do we want to capture the exception still here in this log call like the following.  (I might be out of my context here, so no big deal if that doesn't make sense)
```suggestion
            logger.error(String.format("Error loading Geospatial data from %s", shapeFileUrl), ex);
```

**Review by ckittel — CHANGES_REQUESTED** [review]
Just a couple questions

File: README.md (hunk 1 above)
Comment (hallihan) [reply README.md:19]: Good catch, that link is indeed broken on the version from reference-architectures.  It looks like there's other relative links that are broken in both repos.  The content page at https://github.com/MicrosoftDocs/architecture-center-pr/blob/master/docs/reference-architectures/data/stream-processing-databricks.md seems to have come from the same source at a point in the past, but has many divergent updates.  I'll do a scrub of all the links.

File: README.md (hunk 2 above)
Comment (hallihan) [reply README.md:252]: We don't actually use Python for this, I was just taking from the most recently updated repo.  I'll confirm the version compatibility for 6.4 and see if I can leave this more current.

File: azure/AzureDataBricksJob/src/main/java/com/microsoft/pnp/GeoFinder.java (hunk 3 above)
Comment (hallihan) [reply azure/AzureDataBricksJob/src/main/java/com/microsoft/pnp/GeoFinder.java:81]: That would provide more data in the failure instance, I'll run a quick test and include the change assuming it all works as expected.

File: README.md (hunk 2 above)
Comment (ckittel) [reply README.md:252]: this wasn't a commented about the python line (just needed to pick a nearby line), it was about being rolled back to 5.5 vs what was in there, 6.4

File: README.md (hunk 1 above)
Comment (hallihan) [reply README.md]: Links fixed, it was just this one and one other relative link in a paragraph above.

File: README.md (hunk 2 above)
Comment (hallihan) [reply README.md]: Changed the version to 6.6 which is the latest we can support without moving to Spark 3. (Tested and it worked ok)

File: azure/AzureDataBricksJob/src/main/java/com/microsoft/pnp/GeoFinder.java (hunk 3 above)
Comment (hallihan) [reply azure/AzureDataBricksJob/src/main/java/com/microsoft/pnp/GeoFinder.java]: Confirmed that it builds and runs.  I didn't have any exceptions so I didn't get to see it in action, but should provide better output in the case of an error.

**Review by ckittel — APPROVED** [review]
💯

**Pull request merged**